src/ansys/motorcad/core/rpc_client_core.py

```diff
@@ -428,12 +428,18 @@ def _find_free_motor_cad(self):
                 )
 
     def ensure_version_at_least(self, required_version):
+        """Ensure that the Motor-CAD version is later or equal to required version."""
+        if not self.check_version_at_least(required_version):
+            raise MotorCADError(
+                "This function requires Motor-CAD version: " + required_version + " or later"
+            )
+
+    def check_version_at_least(self, required_version):
         """Check that the Motor-CAD version is later or equal to required version."""
```

Comment:
```suggestion
        """Check if the Motor-CAD version is later or equal to required version."""
```
Same here about not having "Parameters" and Returns" sections.